src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java

```diff
@@ -223,6 +223,8 @@ static class OptionsValues {
         boolean ignoreSigning = false;
         boolean bindServices = false;
         boolean suggestProviders = false;
+
+        int compressLevel;
```

Comment:
It looks like this isn't being used anywhere?

Reply:
Thanks. Removed.